apps.json

```diff
@@ -52,5 +52,9 @@
     "name": "DTube",
     "homepage": "https://d.tube",
     "url_scheme": "https://d.tube/v/{username}/{permlink}"
+  },
+  "beempy": {
+    "name": "beempy",
+    "homepage": "https://github.com/holgern/beem"
```

Comment:
it needs `url_scheme`, otherwise some apps which already utilize script will break